{
  "comments": [
    {
      "key": {
        "uuid": "baa041b7_d79e0962",
        "filename": "README.md",
        "patchSetId": 13
      },
      "lineNbr": 9,
      "author": {
        "id": 1000777
      },
      "writtenOn": "2016-05-31T11:21:38Z",
      "side": 1,
      "message": "not sure something is missing here, \"on which\" maybe ?",
      "revId": "eff7d721a829d2eac0ccfa8092c138513a0c3fe0",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "baa041b7_7a0a848b",
        "filename": "README.md",
        "patchSetId": 13
      },
      "lineNbr": 9,
      "author": {
        "id": 1006607
      },
      "writtenOn": "2016-05-31T14:19:32Z",
      "side": 1,
      "message": "Done",
      "parentUuid": "baa041b7_d79e0962",
      "revId": "eff7d721a829d2eac0ccfa8092c138513a0c3fe0",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "baa041b7_3ae99cb7",
        "filename": "templates/delete-node.j2",
        "patchSetId": 13
      },
      "lineNbr": 4,
      "author": {
        "id": 1000779
      },
      "writtenOn": "2016-05-31T14:21:16Z",
      "side": 1,
      "message": "Would this output go to a log somewhere?",
      "revId": "eff7d721a829d2eac0ccfa8092c138513a0c3fe0",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "baa041b7_9ae0a8ce",
        "filename": "templates/scale-deployment.j2",
        "patchSetId": 13
      },
      "lineNbr": 28,
      "author": {
        "id": 1000779
      },
      "writtenOn": "2016-05-31T14:21:16Z",
      "side": 1,
      "message": "Should this be in some general debug role atht gets called on stack failure always - if a debug flag is set. Not just useful for scale failures",
      "revId": "eff7d721a829d2eac0ccfa8092c138513a0c3fe0",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    }
  ]
}